oaknational/Oak-Web-Application
# ENG-481 build(image data queries): update to filter assets with full data

## Description

- filters assets by published + existence of necessary properties such as url/slug
- this should further prevent issues we saw before of incomplete assets causing issues in builds

## How to test

1. Go to https://deploy-preview-1943--oak-web-application.netlify.thenational.academy
2. Click around
3. You should see no change (and percy shouldn't either!)
